.gitignore

```diff
@@ -59,3 +59,6 @@ specialexport.json
 
 # Custom CSS file
 web/custom.css
+
+# Local SSL certificates
+*.pem
```

Comment:
should this really be part of the wallabag gitignore? better add it in a global gitignore locally :)
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
I was wondering too but I was thinking that if an other developer setup wallabag to run locally, maybe pem files will be defined in the root folder?

Reply:
> if an other developer setup wallabag to run locally

and configure HTTPS, so multiple steps that reduce the chances that this happen.
to me either wallabag provides HTTPS support directly, either it's up to the dev that manually configure HTTPS to ignore any extra file.